tests/test_model.py

```diff
@@ -78,6 +79,12 @@ def test_get_model_context_manager(trained_rasa_model: str):
     assert not os.path.exists(unpacked)
 
 
+@pytest.mark.parametrize("model_path", ["foobar", "rasa", "README.md", None])
+def test_verify_model_path_exception(model_path: Optional[Text]):
```

Comment:
If you write this test then you probably also want one which checks with valid model files, don't you?